js/scripts.js

```diff
@@ -0,0 +1,19 @@
+function drawTree(i){
+    
+	for(i = 1; i <= 5; i++){
```

Comment:
add space after `for` and add declaration for `i`, var `i` should start with `0`